updated php-soap instructions for sharepoint. (#233)
owncloud/docs-server: modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc

The plugin uses the Simple Object Access Protocol (SOAP) and WebDAV for
the uploads and downloads to talk to SharePoint servers. Your ownCloud

> server must have `php-seclib` and `php-soap` or `php7-soap` installed; and enabled soap as a php module.

Author: not sure about php-seclib.
The actual package name is php-phpseclib but then it still does not seem to work. 
More investigation needed.

> server must have `php-seclib` and `php-soap` or `php7-soap` installed; and enabled soap as a php module.
> A webserver restart may be needed after changing php modules.

Reviewer: ```suggestion
server must have the php modules `php-seclib` and `php-soap` installed and enabled, a webserver restart may be needed after adding/changing php modules.
```

> and `phpenmod soap` enabled. A webserver restart may be needed after adding/changing php modules.

Reviewer: ```suggestion
and the SOAP module enabled with `phpenmod soap`. A webserver restart may be required after adding/changing php modules.
```
